src/iperf_api.c

```diff
@@ -2381,7 +2400,8 @@ iperf_free_test(struct iperf_test *test)
         SLIST_REMOVE_HEAD(&test->streams, streams);
         iperf_free_stream(sp);
     }
-
+    free(test->remote_congestion_used);
+    test->remote_congestion_used = NULL;
```

Comment:
in actual master this is already handled with lines 2413-2414
`if (test->remote_congestion_used)`
`	free(test->remote_congestion_used);`